scripts/vim-patch.sh

```diff
@@ -26,7 +26,28 @@ usage() {
   echo "The default is '${VIM_SOURCE_DIR_DEFAULT}'."
 }
 
+# Checks if a program is in the user's PATH, and is executable.
+check_executable() {
+  if [[ ! -x $(command -v "${1}") ]]; then
+    >&2 echo "${BASENAME}: '${1}' not found in PATH or not executable."
+    >&2 echo "Aborting execution."
```

Comment:
I don't think this line is of much use, the above line is pretty clearly an error.